src/endpoints/custodians/domain/use-cases/get-custodians.use-case.ts

```diff
@@ -31,7 +31,7 @@ export class GetCustodiansUseCase
 			limit,
 		});
 
-		if (Array.isArray(rows) && rows.length === 0) {
```

Comment:
Can we specifically check for the failure type here?

To be sure that we actually were able to make connection and query but there is no data.